clients/cli/src/main/java/org/apache/gravitino/cli/commands/AuditCommand.java

```diff
@@ -0,0 +1,37 @@
+package org.apache.gravitino.cli.commands;
```

Comment:
You'll need a license header on this file or the rat compile check will fail

Reply:
Add it.